src/settings/LostItemFeePolicy/components/ViewSections/LostItemFeeSection/LostItemFeeSection.js

```diff
@@ -44,8 +44,12 @@ const LostItemFeeSection = (props) => {
     : '-';
 
   return (
-    <div data-test-lost-item-policy-detail-fee-section>
+    <div
+      data-testid="viewLostItemFeeSectionTestId"
```

Comment:
nit: Maybe let us not use postfix `TestId`. We already have `data-testid` attribute which says that it is test id

Reply:
Good idea, agree, thanks